sirepo/const.py

```diff
@@ -16,6 +16,8 @@
 #: where template resources and template non-sim user files live
 LIB_DIR = "lib"
 
+LOCALHOST_FQDN = "localhost.localdomain"
```

Comment:
In a separate issue, move this to pykern.const where the other localhost stuff is.

Reply:
https://github.com/radiasoft/pykern/issues/555